lit_tests/kernel/wave/location.py

```diff
@@ -222,3 +222,68 @@ def test_stack_trace_dedup(
     # CHECK-SAME:  loc(#[[loc_addf:.+]])
     # CHECK:       #[[loc_load]] = loc(callsite(#{{.*}} at #[[loc_parent:.+]])
     # CHECK:       #[[loc_addf]] = loc(callsite(#{{.*}} at #[[loc_parent]])
+
+
+@run_test
+def test_reduce_op_location():
+    K = tkl.sym.K
+    ELEMS_PER_THREAD = tkl.sym.ELEMS_PER_THREAD
+    constraints: list[tkw.Constraint] = [
+        tkw.HardwareConstraint(
+            threads_per_wave=64,
+            vector_shapes={M: 1, K: 128},
+        )
+    ]
+    constraints += [tkw.WorkgroupConstraint(M, BLOCK_M, 1)]
+    constraints += [tkw.WaveConstraint(M, BLOCK_M)]
+
+    subs = {
+        M: 256,
+        K: 128,
+        BLOCK_M: 1,
+        ELEMS_PER_THREAD: 2,
```

Comment:
Just curious, if we do not pass elems per thread here, is the compiler able to deduce it implicitly?